test/release/examples/benchmarks/shootout/Makefile

```diff
@@ -47,10 +47,10 @@ nbody: nbody.chpl
 	$(CHPL) -o $@ $(CHPL_FLAGS) --no-warnings $<
 
 pidigits: pidigits.chpl
-	$(CHPL) -o $@ $(CHPL_FLAGS) $<
+	@if [ -z "$$CHPL_GMP" ]; then $(CHPL) -o $@ $(CHPL_FLAGS) $<; fi
```

Comment:
I don't think this is quite what you want since we (and users) often won't have GMP or REGEXP explicitly set. I think you need to check for `$CHPL_HOME/util/chplenv/chpl_gmp.py != "none"`